### PR TITLE
Implement pipeline support for TAZ unit tests

### PR DESCRIPTION
**Pull request by dennis-behm**
This PR is dropping the support for IDZ Zunit test setup, and introduces TAZ Unit test support in zAppBuild.

### CODE REVIEW

File: build-conf/datasets.properties [hunk 1]
```diff
@@ -69,8 +69,8 @@ SEQAMOD=
 # Optional IDz Load Library. Example: FEL.V14R0M0.SFELLOAD
 SFELLOAD=
 
-# Optional IDZ zUnit / WAZI VTP library containing necessary copybooks. Example : FEL.V14R2.SBZUSAMP
-SBZUSAMP=
+# Optional IBM Debug Tool library necessary TAZ copybooks. Example : DEBUG.V16.SEQASAMP
```
Comment (M-DLB) [inline build-conf/datasets.properties:72]: Is this value optional?

File: languages/Cobol.groovy [hunk 2]
```diff
@@ -246,7 +246,7 @@ def createCompileCommand(String buildFile, LogicalFile logicalFile, String membe
 		
 	// add additional zunit libraries
 	if (isZUnitTestCase)
-	compile.dd(new DDStatement().dsn(props.SBZUSAMP).options("shr"))
+	compile.dd(new DDStatement().dsn(props.SEQASAMP).options("shr"))
```
Comment (M-DLB) [inline languages/Cobol.groovy:249]: In there an indentation issue here?

File: languages/TazUnitTest.groovy [hunk 3]
```diff
@@ -54,55 +55,55 @@ int currentBuildFileNumber = 1
...
 
 	jcl += """\
 //REPLAY.BZURPT DD DISP=SHR,
-// DSN=${props.zunit_bzureportPDS}(${member})
+// DSN=${props.tazunittest_bzureportPDS}(${member})
 """
 
-// Add parms for bzupplay proc / zUnit Runner
-zunitDebugParm = props.getFileProperty('zunit_userDebugSessionTestParm', buildFile)
+	// Add debugger paramters
```
Comment (M-DLB) [inline languages/TazUnitTest.groovy:82]:
```suggestion
	// Add debugger parameters
```

File: languages/TazUnitTest.groovy [hunk 4]
```diff
@@ -173,46 +171,29 @@ zunitDebugParm = props.getFileProperty('zunit_userDebugSessionTestParm', buildFi
...
+			// Store Report in Workspace
+			new CopyToHFS().dataset(props.tazunittest_bzureportPDS).member(member).file(reportLogFile).copyMode(DBBConstants.CopyMode.valueOf("BINARY")).append(false).copy()
+			// print warning and report
+			println warningMsg
+			printReport(reportLogFile)
+			buildUtils.updateBuildResult(warningMsg:warningMsg,logs:["${member}_tazunittest.log":logFile])
+		} else { // rc > props.tazunittest_maxWarnRC.toInteger()
 			props.error = "true"
-			String errorMsg = "*!  zUnit Test Job ${zUnitRunJCL.submittedJobId} failed with ${zUnitRunJCL.maxRC}"
+			String errorMsg = "*! The zunit test failed with RC=($rc) for $buildFile "
```
Comment (M-DLB) [inline languages/TazUnitTest.groovy:191]:
```suggestion
			String errorMsg = "*! The TAZ Unit Test failed with RC=$rc for '$buildFile'"
```

File: build-conf/README.md [hunk 5]
```diff
@@ -30,7 +30,7 @@ SCSQLOAD | MQ Load Library. Example: CSQ.V9R1M0.SCSQLOAD
 SDSNLOAD | DB2 Load Library. Example: DB2.V9R1M0.SDSNLOAD
 SDSNEXIT | DB2 Exit Library. Example: DBC0CFG.SDSNEXIT
 SFELLOAD | Optional IDz Load Library. Example: FEL.V14R0M0.SFELLOAD
-SBZUSAMP | Optional z/OS Dynamic Test Runner IDz zUnit / WAZI VTP library containing necessary copybooks. Example : FEL.V14R2.SBZUSAMP
+SEQASAMP | Optional IBM Debug Tool library containing necessary TAZ copybooks. Example : DEBUG.V16.SEQASAMP
```
Comment (M-DLB) [inline build-conf/README.md:33]: Is it optional?

File: build-conf/datasets.properties (hunk 1 above)
Comment (M-DLB) [inline build-conf/datasets.properties:72]:
```suggestion
# Optional IBM Debug Tool library containing necessary TAZ copybooks. Example : DEBUG.V16.SEQASAMP
```

**Review by M-DLB — COMMENTED** [review]
Few comments

File: build-conf/datasets.properties (hunk 1 above)
Comment (dennis-behm) [reply build-conf/datasets.properties:72]: Generally it is optional, but obviously required when the user is leveraging TAZ

File: build-conf/README.md (hunk 5 above)
Comment (dennis-behm) [reply build-conf/README.md:33]: Required for TAZ, but optional for a normal Cobol build.

**Comment by dennis-behm** [comment]
@M-DLB I rebased/merged `upstream/develop` and resolved some merge conflicts. Your comments should be addressed as well. 

Please see the attached a build run with outputs generated with TAZ 1.0.0.3.

[pr561.log](https://github.com/user-attachments/files/17662283/pr561.log)

**Comment by dennis-behm** [comment]
@M-DLB Introduced a dedicated job card property for the taz unit tests, and uplifted a couple of properties in the with the recent commits. 

Can you please have another check? Thanks

**Review by M-DLB — APPROVED** [review]
Thank you @dennis-behm

**Pull request merged**